app/frontend/src/routes.ts

```diff
@@ -1,3 +1,4 @@
+export const authRoute = "/auth";
```

Comment:
Maybe this should be / and the main logged in page should be /dashboard?

Reply:
Or maybe both should be / but rendered conditionally if user is logged in or not?

Reply:
Sounds good